src/menu/compas-open-plugin.ts

```diff
@@ -0,0 +1,181 @@
+import {
+  css,
+  html,
+  LitElement,
+  query,
+  TemplateResult,
+  property,
+  state,
+} from 'lit-element';
+import { translate, registerTranslateConfig, use } from 'lit-translate';
+import { loader } from '../translations/loader.js';
+import { Dialog } from '@material/mwc-dialog';
+import { newOpenEvent } from '@openscd/open-scd-core';
+import {
+  CompasSclDataService,
+  SDS_NAMESPACE,
+} from '../service/CompasSclDataService.js';
+
+import { DocRetrievedEvent } from '../compas-ui/compas-open.js';
+import '../compas-ui/compas-open.js';
+import '../compas-ui/helpers/compas-scl-list.js';
+import '../compas-ui/helpers/compas-scl-type-list.js';
+import '@material/mwc-button';
+import '@material/mwc-icon-button';
+import '@material/mwc-icon';
+import '@material/mwc-dialog';
+import { buildDocName } from '../to-be-made-a-dependency/foundation.js';
+import { SclSelectedEvent } from '../compas-ui/helpers/compas-scl-list.js';
+import { TypeSelectedEvent } from '../compas-ui/helpers/compas-scl-type-list.js';
+export default class CompasOpenMenuPlugin extends LitElement {
+  @query('mwc-dialog#compas-open-dlg')
```

Comment:
can tsdoc be added to these properties?

https://tsdoc.org/

Reply:
☑️